src/backend/tracker.js

```diff
@@ -67,6 +48,25 @@ async function trackerFunc() {
 			}
 		}
 	}
+	const serializedData = serializeTabRepository(repo);

```

Comment:
isn't serializing the data also a part of the util.js file cause technically it is a utility function